test/post-911-gib-status/components/PrintPage.unit.spec.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+import { PrintPage } from '../../../src/js/post-911-gib-status/containers/PrintPage';
+
+describe('<PrintPage/>', () => {
+  const pushSpy = sinon.spy();
+  const defaultProps = {
+    router: { push: pushSpy },
+    enrollmentData: {}
+  };
+
+  it('should render', () => {
+    const wrapper = shallow(<PrintPage {...defaultProps}/>, { disableLifecycleMethods: true });
```

Comment:
I think you told me the other day, but why disable the lifecycle methods on these?

Reply:
If they're not disabled, they try to do DOMy things, which breaks the tests, because the lifecycle method in question tries to select header & footer elements directly from the DOM. These don't exist in the context of the unit test, or really the React app at all, since they live in the megamenu.